pkg/test/common.go

```diff
@@ -167,6 +167,43 @@ func CopyFiles(sourceDir, destDir string) error {
 	return nil
 }
 
+type Controller interface {
+	Run(ctx context.Context) error
+	Shutdown()
+	GetPort() int
+}
+
+type ControllerStruct struct {
+	c Controller
+}
+
+func (c *ControllerStruct) StartServer() {
+	// this blocks
```

Comment:
This comment is no longer relevant?

Reply:
I added the comment back in controller_test, just before cm.StartServer()